molgenis-mail/src/main/java/org/molgenis/mail/MailSettings.java

```diff
@@ -0,0 +1,20 @@
+package org.molgenis.mail;
+
+interface MailSettings
+{
+	String getHost();
```

Comment:
You mean, can we configure a running CentOS sendmail server using these parameters?